src/screens/dashboard/dashboard.tsx

```diff
@@ -209,6 +239,77 @@ export class DashboardScreenComponent extends React.Component<
             // NavigationEvents is not enough for the web in order to get balances
             this.onFocus();
         }
+
+        if (
+            this.props.walletId !== prevProps.walletId &&
+            isFeatureActive(RemoteFeature.NOTIF_CENTER)
+        ) {
+            this.hasUnseenNotifications();
+        }
+    }
+
+    private async hasUnseenNotifications() {
+        try {
+            const http = new HttpClient(CONFIG.notificationCenter.hasUnseenNotifs);
```

Comment:
since we use redux, we should do all http calls in actions, this way the logic of the screen remains simple, and also actions are reusable...